frontend/src/features/projects/common/forms/ProjectDraftForm.tsx

```diff
@@ -55,7 +55,7 @@ const ProjectDraftForm = ({
         />
       </Form.Row>
       <Form.Row>
-        <Input
+        <TextArea
```

Comment:
Made sense for the description to be a textarea to allow a max of 1000 characters